modules/about-update-channels.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * installation assemblies
+
+:_content-type: CONCEPT
+[id="about-update-channels_{context}"]
+= About update channels
```

Comment:
This title is probably fine, but was thinking maybe a little generic (i.e., sounds like it could be about Operator update channels in general, but it actually goes further into OADP usage specifics). Maybe "About update channels for OADP" or "About OADP update channels"? Matches the assembly's title "About installing OADP" a little more.

Reply:
Changes made.